app/mutations/FundingParameter/updateFundingParameterFormChange.ts

```diff
@@ -4,8 +4,7 @@ import useDebouncedMutation from "mutations/useDebouncedMutation";
 
 /**
  *
- * Specific mutation for a form_change for a project.
- * Includes whether the suggested proposal reference is unique in the system.
+ * Specific mutation for a form_change for a funding agreement.
```

Comment:
Not sure why we weren't using this mutation before, but saved me having to make it

Reply:
Will we not need the generic `updateFormChange` once the funding page is refactored to a single form as noted in [insert_json_schema_form_data_separate_EP_and_IA_funding_schemas.sql](https://github.com/bcgov/cas-cif/pull/1400/files#diff-b21441d8fc81741690b9066a831d732af88890a3f8f80ffd8cd82fb7a1164ddc)?

Reply:
I think we'll continue to use custom update form change functions even after the refactor. Whenever a form needs to update things that aren't in `updateFormChange` (so in this case, some funding-specific computed columns), we can use a custom function so we're not grabbing those extra things for other forms that use `updateFormChange` but don't need the extras. The milestone form has been refactored and still has a custom update.

Reply:
The only component using the generic `updateFormChange` is the funding parameter. It's also used in the FormPageFactory. I'm wondering if it's a better pattern to use custom mutations for specific components so changes are decoupled.